doc/api/http2.md

```diff
@@ -1725,7 +1725,7 @@ those to lower-case (e.g. `content-type`) upon transmission.
 
 Header field-names *must only* contain one or more of the following ASCII
 characters: `a`-`z`, `A`-`Z`, `0`-`9`, `!`, `#`, `$`, `%`, `&`, `'`, `*`, `+`,
-`-`, `.`, `^`, `_`, `` (backtick), `|`, and `~`.
+`-`, `.`, `^`, `_`, `` (backtick)`, `|`, and `~`.
```

Comment:
I think this maybe would be better with `(backtick)` in plain (non-code) text? i.e. ``` `^`, `_`, `` ` `` (backtick), `|`, …```